app/controllers/kudos_controller.rb

```diff
@@ -4,6 +4,7 @@ class KudosController < ApplicationController
   before_action :set_kudo, only: %i[show edit update destroy]
   before_action :authenticate_employee!
   before_action :check_kudo_giver, only: %i[edit update destroy]
+  rescue_from Pundit::NotAuthorizedError, with: :timelimit_for_edit_delete
```

Comment:
well done! what if another resource will require authorization? Probably you will need to repeat the rescue_from which is not ideal :).

Reply:
to not repeat rescue_form according to pundit documentation we can use `config.action_dispatch.rescue_responses["Pundit::NotAuthorizedError"] = :forbidden` witch returns error 403 when not auth